bot.py

```diff
@@ -42,10 +42,22 @@ def main():
 
     check_tsschecker()
 
-    intents = discord.Intents.default()
-    intents.members = True
-
-    client = commands.AutoShardedBot(command_prefix=get_prefix, help_command=None, intents=intents)
+    mentions = discord.AllowedMentions(
+        **dict.fromkeys([
```

Comment:
I'd prefer if `roles` and `everyone` were just passed as args into `AllowedMentions`, rather then using a dict, as it's a bit unnecessary.

Reply:
While it uses more lines, it's generally more readable; it circumvents the need to scroll on your IDE to the right to see the entire line. This has already been removed though.